api/organisations/tasks.py

```diff
@@ -41,3 +48,16 @@ def update_organisation_subscription_information_cache():
     subscription_info_cache.update_caches(
         (SubscriptionCacheEntity.CHARGEBEE, SubscriptionCacheEntity.INFLUX)
     )
+
+
+@register_recurring_task(
+    run_every=timedelta(hours=12),
+)
```

Comment:
Lots of experience with this type of thing for the erasure work at Clubhouse taught me not to rely on task scheduling far in the future, too many things can go wrong. Plus, as you say, this decouples criteria and action so it would work for Django admin as well.